src/controller/org.controller/index.js

```diff
@@ -322,14 +317,13 @@ router.get('/org/:shortname/users',
   /*
   #swagger.tags = ['Users']
   #swagger.operationId = 'userOrgAll'
-  #swagger.summary = "Retrieve all user records specified by the short name of the organization"
+  #swagger.summary = "Retrieves all user information in an organization for the speicifed short name (accessible to all users)"
   #swagger.description = "
         <h2>Access Control</h2>
-        <p>No roles needed to access the endpoint</p>
+        <p>All registered users can access this endpoint</p>
         <h2>Expected Behavior</h2>
-        <p><b>Secretariat:</b> Can see the information of all users of any Organization</p>
-        <p><b>Admin User:</b> Can only see the information of users that belongs to the same Organization</p>
-        <p><b>Regular User:</b> Can only see the information of users that belongs to the same Organization</p>"
+        <p><b>Regular and Admin Users:</b> Retrieves information of users in the same organization</p>
```

Comment:
Make this Regular, CNA & Admin Users. Minor nit - I would say retrieves information FOR rather than OF